stac_fastapi/elasticsearch/tests/api/test_api.py

```diff
@@ -123,238 +116,146 @@ def test_app_context_extension(load_test_data, app_client, es_transactions, es_c
     assert "context" in resp_json
     assert resp_json["context"]["returned"] == resp_json["context"]["matched"] == 1
 
-    es_transactions.delete_collection(collection["id"], request=MockStarletteRequest)
```

Comment:
I removed tons of these calls, as it's handled by `ctx` now

Reply:
Awesome